robot/Cumulus/tests/browser/recurring_donations/zedit_effective_date_erd_record.robot

```diff
@@ -0,0 +1,127 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/RecurringDonationsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             DateTime
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Enable RD2
+#Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+
+Setup Test Data
+
+        ${effective_date_initial} =           Get Current Date      result_format=%-m/%-d/%Y
+        ${date} =           Get Current Date
+        ${DATE}=            Add Time To Date      ${date}  30 days      result_format=%Y-%m-%d %H:%M:%S.%f
+        ${date_to_update} =   Convert Date                   ${DATE}                         result_format=%Y-%m-%d
```

Comment:
You can instead write this as:
${date_to_update} =       Get current date       result_format=%Y-%m-%d           increment=30 days

I see that you are not using {DATE} variable anywhere else, can you remove it?